sklearn/utils/multiclass.py

```diff
@@ -227,7 +200,7 @@ def is_multilabel(y):
     True
 
     """
-    return is_label_indicator_matrix(y) or is_sequence_of_sequences(y)
```

Comment:
Unless we plan to support additional formats for the multilabel case, could this `is_multilabel` be safely replaced everywhere by `is_label_indicator_matrix`?